src/components/widgets/EventWidget.tsx

```diff
@@ -12,14 +12,15 @@ import { WidgetProps } from '../../types';
 
 import { DefaultWidget } from './DefaultWidget';
 
+const currentDate = moment().format('YYYY-MM-DD');
+
```

Comment:
How did you benefit from moving it here?

Reply:
i thought to move it outside of the component as it has nothing to do with the component and its lifecycle, it does not depend on anything. it is like a constant value for each day, which do not need to be recalculated on every mount or re-renderings. the variable can be created and "stored" more globally in the code base for its usage in the EventWidget component.